chapters/ch06/3.spell-correction.ipynb

```diff
@@ -428,6 +413,14 @@
        "      <td>1</td>\n",
        "    </tr>\n",
        "    <tr>\n",
+       "      <th>61</th>\n",
+       "      <td>laptopa</td>\n",
+       "      <td>laptops</td>\n",
+       "      <td>6</td>\n",
+       "      <td>5565</td>\n",
```

Comment:
This looks very different than what was there before (5565 vs 14119). I wonder why?